source/installation.txt

```diff
@@ -28,36 +28,54 @@ Installation Guides
 Supported Platforms
 -------------------
 
-|bi-short| supports the following 32-bit and 64-bit operating
+|bi-short| supports the 32-bit and 64-bit versions of the following
+Windows editions:
+
+.. list-table::
+   :widths: 40 60
+   :stub-columns: 1
+   :header-rows: 1
+
+   * - Windows Edition
+     - Versions
+
+   * - Windows
+     - 7 or later
+
+   * - Windows Server
+     - 2008 R2 or later
+
+|bi-short| supports the 64-bit versions of following the operating
 systems:
 
 .. list-table::
-   :widths: 50 25 25
+   :widths: 40 60
    :stub-columns: 1
+   :header-rows: 1
 
-   * - Windows (32- and 64-bit)
-     - Windows 7 or later
-     - Windows Server 2008 R2 or later
+   * - Operating System
+     - Versions
 
-   * - macOS (64-bit)
+   * - macOS
      - 10.10 or later
```

Comment:
nit: the mixed bullet/no-bullet approach make the columns line up in a weird way. Maybe make all of these use bulleted lists even if they only have one version listed?

Reply:
Agreed. Fixed.